catalog/spatial/csw/spatial-csw-source-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/source/AbstractCswSource.java

```diff
@@ -144,7 +168,7 @@
  * services.
  */
 public abstract class AbstractCswSource extends MaskableImpl
-        implements FederatedSource, ConnectedSource, ConfiguredService {
+        implements CatalogStore, FederatedSource, ConnectedSource, ConfiguredService {
```

Comment:
I don't think it is the typical use case for CswSources to implement `CatalogStore` this should not be part of the core implementation.